src/mlpack/core/optimizers/cmaes/random.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <ctime>
+#include <cmath>
+
+/**
+ * @class Random
+ * A pseudo random number generator.
+ */
+template<typename T>
+class Random
```

Comment:
i dont know @zoq randu just works nicely but when i replace it with rand.gauss() function 
it creates very wrong results. Ive tried this before but no success.
can we keep the random class ?